### ISSUE TITLE
Make models show correct amounts

### ISSUE THREAD

**Issue by Nekuromu** [Goldbar97/mods#28]
![image](https://github.com/Goldbar97/mods/assets/30809120/2ceccc32-c2d3-4e7d-a226-535878223410)

**Comment by Goldbar97** [comment]
That's exactly what we've concerned about?

**Comment by Nekuromu** [comment]
yup, it's something we already discussed; those edits to the models and I still plan to do them! It's just good to have this on hand as a direct issue made. So I took the opportunity to add it!
